README.md

```diff
@@ -1,70 +1,60 @@
-# Getting Started with Create React App
+# Math-Magicians
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+"Math-Magicians" is an app that helps people solve numerical challenges. As a user, you can use the app to do math calculations. The app has a beautiful interface with interactive features.
 
-## Available Scripts
 
-In the project directory, you can run:
+## Built With
 
-### `npm start`
+- React
+- JavaScript
+- CSS3
+- WebPack
+- JSX
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+## Live Demo 
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+[Live Demo App]()
```

Comment:
- [ ] Please, consider fixing this broken link: you can remove this link for current milestone 👍